<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloaderDialog_V2</class>
 <widget class="QDialog" name="DownloaderDialog_V2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download with Pause and Resume</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_OpenSaveFile">
     <property name="text">
      <string>Select File...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_DownloadPath">
     <property name="text">
      <string>Download To:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_DownloadPath</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_DownloadPath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser_Console"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_Close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_DownloadURL">
     <property name="text">
      <string>Download Url</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_DownloadURL</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_DownloadURL">
     <property name="inputMethodHints">
      <set>Qt::ImhUrlCharactersOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_Download">
     <property name="text">
      <string>Download</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_Status">
     <property name="title">
      <string>Download Status</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Pause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Resume">
        <property name="text">
         <string>Resume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancle">
        <property name="text">
         <string>Cancle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>DownloaderDialog_V2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
